config/networking-wildcardcert.yaml

```diff
@@ -0,0 +1,69 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: networking-wildcardcert
+  namespace: knative-serving
+  labels:
+    serving.knative.dev/release: devel
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: networking-wildcardcert
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
+      labels:
+        app: networking-wildcardcert
+    spec:
+      serviceAccountName: controller
+      containers:
+      - name: networking-wildcardcert
+        # This is the Go import path for the binary that is containerized
+        # and substituted here.
+        image: knative.dev/serving/cmd/networking/wildcardcert
```

Comment:
Could we put the main.go file into `knative.dev/serving/cmd/networking/ns` if we agree that this controller could be used for the future namespace related work?